<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="NumberEdit" name="editPasswd">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>361</width>
     <height>45</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>45</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>45</height>
    </size>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhPreferNumbers|Qt::ImhSensitiveData</set>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>81</width>
     <height>45</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>45</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>45</height>
    </size>
   </property>
   <property name="text">
    <string>密码:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancle">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>240</y>
     <width>110</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>240</y>
     <width>110</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>491</width>
     <height>47</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QCheckBox" name="chboxOperator">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>45</height>
       </size>
      </property>
      <property name="text">
       <string>操作员</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <property name="autoExclusive">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="chboxAdmin">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>45</height>
       </size>
      </property>
      <property name="text">
       <string>管理员</string>
      </property>
      <property name="autoExclusive">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="chboxAdvance">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>45</height>
       </size>
      </property>
      <property name="text">
       <string>高级</string>
      </property>
      <property name="autoExclusive">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NumberEdit</class>
   <extends>QLineEdit</extends>
   <header>customedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
